Client/Frontend/Widgets/AutocompleteTextField.swift

```diff
@@ -371,7 +371,8 @@ extension AutocompleteTextField: ThemeApplicable, PrivateModeUI {
 
         backgroundColor = theme.colors.layer3
         textColor = theme.colors.textPrimary
-        tintColor = isPrivateMode ? theme.colors.layerAccentPrivateNonOpaque : theme.colors.layerAccentNonOpaque
+        let highlightColor = isPrivateMode ? theme.colors.layerAccentPrivateNonOpaque : theme.colors.layerAccentNonOpaque
+        tintColor = highlightColor
```

Comment:
FYI I am changing that tintColor in https://github.com/mozilla-mobile/firefox-ios/pull/15203 after conversation on that task